karrot/history/serializers.py

```diff
@@ -18,6 +18,9 @@ class Meta:
             'place',
             'users',
             'payload',
+            'message',
+            'before',
```

Comment:
Did you check if the values in `before` and `after` are always good to return? They contain all fields of the related model,

Reply:
Ah good point. I did not check. Maybe we can run it through the serialiser. Hmm I'll have a think.

Another way would be only returning it for my new types, adding a bit of tech debt until we can rationalize the history data.

Reply:
It already used the serializer to write into the `after` field (I presume others do that too), but I decided to make it so only the new specific history types will return the `after` field (and the `before` field was actually unused in my frontend code, so I removed that entirely).